Documentation/gettingstarted/clustermesh/clustermesh.rst

```diff
@@ -10,6 +10,10 @@ connecting them together, enable pod-to-pod connectivity across all clusters,
 define global services to load-balance between clusters and enforce security
 policies to restrict access.
 
+Aside from this step-by-step guide, if you would like to watch how Cilium's 
+Clustermesh feature makes it easy to run services backed by pods distributed 
+across multiple Kubernetes clusters, check out this video: `eCHO Episode 41: Cilium Clustermesh <https://www.youtube.com/watch?v=VBOONHW65NU&t=342s>`__.
```

Comment:
I'd remove `this video:`, here and everywhere else.

```suggestion
across multiple Kubernetes clusters, check out `eCHO Episode 41: Cilium Clustermesh <https://www.youtube.com/watch?v=VBOONHW65NU&t=342s>`__.
```

Reply:
Will do so. Thanks for the feedback.